### ISSUE TITLE
Execution Block ID uid://A001/X15a0/X148 Sgr_A_st_ac_03_TM1

### ISSUE THREAD

**Issue by keflavich** [ACES-CMZ/reduction_ACES#232]
Sgr_A_st_ac_03_TM1
[uid://A001/X15a0/X148](https://almascience.org/aq/?result_view=observation&mous=uid://A001/X15a0/X148)

* [x] Observations completed?
* [ ] Delivered?
* [ ] Downloaded? (specify where)
  * [ ] hipergator
  * [ ] hipergator pipeline run

* [ ] [Weblog](https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_ac_03_TM1/html/) unpacked
* [ ] [Weblog](https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_ac_03_TM1/html/) Quality Assessment?
* [ ] Imaging: Continuum
* [ ] Imaging: Lines

  * [ ] SPW25 H13CN
  * [ ] SPW27 H13CO+/SiO
  * [ ] SPW29 HCO+
  * [ ] SPW31 HNCO
  * [ ] SPW33 Cont 1
  * [ ] SPW35 Cont 2

**Comment by d-l-walker** [comment]
Everything looks good with this one. All cubes were binned x2 for size mitigation. I'll leave this one as 'Under QA' until this is resolved. 

One thing to note is that the HCO+ emission is cut off at the lower edge of the window. As far as I can tell, no velocity shift was requested for this SB.

![uid___A001_X15a0_X148 s31_0 Sgr_A_star_sci spw29 mfs I findcont residual autoLowerIntersect meanSpectrum mom0mom8jointMask min min 2 50sigma narrow3 trimauto_max=20 overwriteTrue](https://user-images.githubusercontent.com/13015266/192748950-476b808a-42b7-4bd4-b2c7-d1eef7ce9722.png)
![uid___A001_X15a0_X148 s38_0 Sgr_A_star_sci spw29 cube I iter1 image spectrum](https://user-images.githubusercontent.com/13015266/192748960-e3808962-2b7a-478a-8a67-ce13cf89853e.png)

**Comment by d-l-walker** [comment]
@keflavich now that #287 has been merged, this region will need to be re-cleaned with the updated parameters to get the full cube resolution.

**Comment by keflavich** [comment]
moved all old files to `mitigated_old/` subdirectory.  Re-imaging will trigger soon.

**Comment by d-l-walker** [comment]
@keflavich just checking up on this one, and it looks like SPW 33 has been cleaned at full spectral resolution, but all other SPWs only have the size-mitigated products still?

**Comment by keflavich** [comment]
Could you verify that the unmitigated sizes are appropriately specified in `override_tclean_commands`?

**Comment by d-l-walker** [comment]
Yep, they're all there. I added them in PR #232. But when I checked on globus I noticed that the FITS cubes looked suspiciously small. So I downloaded a couple, and they still have the mitigated number of channels (but SPW 33 is fine).

**Comment by keflavich** [comment]
OK, moved them to `pre_mar1_2023` and triggered imaging.

**Comment by d-l-walker** [comment]
All SPWs have now been imaged at full spectral resolution. The moment maps look good, but I'm going to download and inspect the cubes just to be sure. Assuming all is good, I will mark this as done.

**Comment by djeff1887** [comment]
Cont QA (spw25_27, spw33_35): Seems ok, maybe slight contamination mainly in spw25_27

![image](https://github.com/ACES-CMZ/reduction_ACES/assets/62924149/1fb6fc12-f2b4-4054-b120-b2711ddb9ca5)

**Comment by djeff1887** [comment]
25_27 and 33_35: this seems significantly better than the previous run, the residual is down by a factor of ~2-4, and visually there's only very faint hints of the contamination from before.
![image](https://github.com/ACES-CMZ/reduction_ACES/assets/62924149/976cc675-6c45-42f2-b912-1ded4fb7488f)

![image](https://github.com/ACES-CMZ/reduction_ACES/assets/62924149/ce1217fa-8610-4680-947b-a13fa8e043a6)
